### PR TITLE
BAEL-2360

### PR DESCRIPTION
**Pull request by MajewskiKrzysztof**

### CODE REVIEW

File: spring-boot-mvc/src/main/java/com/baeldung/shedlock/ShedlockScheduler [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.baeldung.shedlock;
```
Comment (jzheaux) [inline spring-boot-mvc/src/main/java/com/baeldung/shedlock/ShedlockScheduler]: Why isn't this a Java file?

File: spring-boot-mvc/pom.xml [hunk 2]
```diff
@@ -72,6 +72,18 @@
             <version>${spring.fox.version}</version>
         </dependency>
 
+        <!-- ShedLock -->
```
Comment (jzheaux) [inline spring-boot-mvc/pom.xml]: It feels weird that this is in `spring-boot-mvc`. Is there a more general one? My concern is this has nothing to do with MVC.

File: spring-boot-mvc/src/main/java/com/baeldung/shedlock/ShedlockScheduler (hunk 1 above)
Comment (jzheaux) [reply spring-boot-mvc/src/main/java/com/baeldung/shedlock/ShedlockScheduler]: Also, can we make sure it matches the article?

**Comment by MajewskiKrzysztof** [comment]
@jzheaux Sorry for that mistake :/
I made changes. I also deleted Application class and created new one: SchedulerConfiguration. I moved all to spring-all project. 
Is it ok now?

**Pull request merged**